featurebyte/migration/service/feature_list.py

```diff
@@ -109,3 +110,67 @@ async def add_relationships_info_and_supported_serving_entity_ids(self) -> None:
             assert "primary_entity_ids" in doc, doc
 
         logger.info("Migrated all records successfully (total: %d)", total_after)
+
+    async def batch_preprocess_document_v6(self, documents: List[Document]) -> List[Document]:
```

Comment:
curious - do you think it might be better to have each migration version live in its own class? might make the different versions slightly more discoverable, and we also won't have to deal with renaming versioning these helper methods too.

Reply:
I don't have strong opinion on this. Sharing on the same class make it easier to reuse certain method. But having a separate class make it a bit less likely to modify previous class. Will move this into a separate class.